src/astarte-client/client.ts

```diff
@@ -21,6 +21,7 @@ import { Socket as PhoenixSocket } from 'phoenix';
 import _ from 'lodash';
 
 import { fromAstartePipelineDTO, toAstartePipelineDTO } from './transforms';
+import * as mocks from './mocks';
```

Comment:
Those are not mocks. Those are actually valid schemas!
We should leave mocks in the testing context.

What about "block definitions"?
```
import { Container ... ... ... } from './blocks/definitions`;
```
Or anything along this line

Reply:
Removed references to `mocks` and I went with `definitions/blocks`